ruby-challenges.rb

```diff
@@ -21,7 +32,13 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowels string
```

Comment:
Great work! FYI, Ruby allows you to use the delete method with passing all of the characters at once. Delete will remove any matches. `.delete "AEIOUaeiou"` When executing this method, that string does not exist as a substring but more like the set of letters A, E, I, O, U, a, e, i, o, u separately.